curriculum/challenges/english/25-front-end-development/workshop-library-manager/67116d7584d0b469b14579c0.md

```diff
@@ -0,0 +1,98 @@
+---
+id: 67116d7584d0b469b14579c0
+title: Step 3
+challengeType: 1
+dashedName: step-3
+---
+
+# --description--
+
+Create another object inside the `library` array with the following properties and values:
+
+| Property | Value   |
+| ----------- | ------- |
+| `title` | `"Atomic Habits"`|
+| `author` | `"James Clear"`|
+| `about` | `"A practical book about discarding bad habits and building good ones"`|
+| `pages` | `320` |
+
+# --hints--
+
+
+You should have an object as the second item of your `library` array.
+
+```js
+assert.isObject(library[1]);
+```
+
+Your object should have a `title` property.
+
+```js
+assert.property(library[1], "title")
+```
+
+Your `title` property should have `"Atomic Habits"` as its value.
+
+```js
+assert.equal(library[1]?.title, "Atomic Habits");
+```
+
+Your object should have an `author` property.
+
+```js
+assert.property(library[1], "author")
+```
+
+Your `author` property should have `"James Clear"` as its value.
+
+```js
+assert.equal(library[1]?.author, "James Clear");
+```
+
+Your object should have an `about` property.
+
+```js
+assert.property(library[1], "about")
+```
+
+Your `about` property should have the value `"A practical book about discarding bad habits and building good ones"`.
+
+```js
+assert.equal(library[1]?.about, "A practical book about discarding bad habits and building good ones");
```

Comment:
For all tests checking for values we should use the propertyVal assert since that is more specific then assert.equal

https://www.chaijs.com/api/assert/#method_propertyval